### PR TITLE
fix: ignore null values.

### PR DESCRIPTION
**Pull request by 37coHiNa**
Related issue: #XXXX

**Description**

A clear and concise description of what the problem was and how this pull request solves it.

This contribution is funded by [Example](https://example.com).

### CODE REVIEW

File: editor/js/libs/ui.three.js [hunk 1]
```diff
@@ -722,18 +722,24 @@ UIPoints2.prototype.constructor = UIPoints2;
...
+	var x = 0, y = 0;
 
-		this.pointsList.add( this.createPointRow( 0, 0 ) );
+	for ( var i = this.pointsUI.length - 1; i >= 0; i -- ) {
 
-	} else {
+		var pointUI = this.pointsUI[ i ];
 
-		var point = this.pointsUI[ this.pointsUI.length - 1 ];
+		if ( pointUI != null ) {
```
Comment (Mugen87) [inline editor/js/libs/ui.three.js:731]: Can you please explain why this change is necessary? 

Is it also required for `UIPoints3`?

Comment (37coHiNa) [reply editor/js/libs/ui.three.js:731]: >Can you please explain why this change is necessary?

The procedure for reproducing the bug I found is as follows.

1. click `Add` ->`Lathe`
2. click `Geometory`
3. click `-`, delete last point

After that, the `+` button will not work and the error will be displayed on the console.

>Is it also required for `UIPoints3`?

I fixed only the part where the error occurred.
Does this change affect other parts?

Comment (37coHiNa) [reply editor/js/libs/ui.three.js:731]:
```javascript
this.pointsUI.splice( idx, 1 );
```

I also wanted to do that.
But this approach causes the problem with the delete button.
Because delete buttons have index when they were created.

Comment (Mugen87) [reply editor/js/libs/ui.three.js:731]: I've made a PR with a different implementation that should solve the issue, too. Do you mind testing #21011?

Comment (37coHiNa) [reply editor/js/libs/ui.three.js:731]: #21011
It’s a better approach than mine.
I have confirmed that any buttons work fine.

Could I close this pull request?

Comment (Mugen87) [reply editor/js/libs/ui.three.js:731]: Thanks for testing!
